manim/__main__.py

```diff
@@ -14,7 +15,17 @@ def exit_early(ctx, param, value):
         sys.exit()
 
 
-console.print(f"Manim Community [green]v{__version__}[/green]")
+latest = requests.get(
+    "https://api.github.com/repos/ManimCommunity/manim/releases/latest"
+)
```

Comment:
I would suggest using PyPi API https://pypi.org/pypi/manim/json as it isn't ratelimited.

Reply:
Also, I would be able to run Manim offline, the error need to be handled.